gulpfile.js

```diff
@@ -805,7 +809,9 @@ function dist() {
         copyAliasExtensions();
       }).then(() => {
         if (argv.fortesting) {
-          return enableLocalTesting(minifiedRuntimeTarget);
+          return enableLocalTesting(minifiedRuntimeTarget).then(() => {
```

Comment:
In person clarification: These are almost instantaneous, so no need to parallelize with `Promise.all`.

Perfect!